spec/filters/grok_spec.rb

```diff
@@ -652,7 +652,7 @@
       pattern_dir = File.join(LogStash::Environment::LOGSTASH_HOME, "patterns")
       @file = Tempfile.new('grok', pattern_dir);
       @file.write('WORD \b[2-5]\b')
-      @file.rewind
+      @file.close
```

Comment:
Tempfile's behavior is why I wrote `Stud::Temporary` btw. We can refactor this to use that instead.